lib/msal-common/src/telemetry/performance/PerformanceClient.ts

```diff
@@ -132,6 +139,13 @@ export abstract class PerformanceClient implements IPerformanceClient {
      */
     abstract setPreQueueTime(eventName: PerformanceEvents, correlationId?: string): void;
 
+    /**
+     * Get integral fields.
+     */
+    getIntFields(): ReadonlySet<string> {
```

Comment:
Do we need this function or can we just directly use `IntFields`?

Reply:
I have added it on purpose to allow overriding integral fields in 1p repo.

Reply:
Maybe add a comment to the code itself so it's clear to the reader.

Reply:
Done